packages/wdio-cucumber-framework/tests/adapter.test.js

```diff
@@ -25,7 +25,14 @@ const wdioReporter = {
 
 const adapterFactory = (cucumberOpts = {}) => new CucumberAdapter(
     '0-2',
-    { cucumberOpts },
+    {
+        cucumberOpts,
+        beforeStep: 'beforeStep',
+        afterStep: 'afterStep',
+        beforeHook: 'beforeHook',
+        afterHook: 'afterHook',
```

Comment:
```suggestion
        afterHook: 'afterHook'
```

Reply:
we can add a linter rule to either have or not the comma in the end. The comma is required in one of my projects, always missing it now